docs/can-canjs/can-api.md

```diff
@@ -277,17 +292,56 @@ console.log(areSomeComplete); //-> false
 ```
 @codepen
 
-<details>
-<summary>Infrastructure APIs</summary>
+</details>
+
+## Typed properties
+
+[can-type] is helpful to define typed properties for models and components by doing the following:
```

Comment:
I think this `can-type` explanation could be improved. I would say something like

> [can-type] can be used to define typed properties in [ObservableObject] and [StacheElement]
> - Types are strict by default, which means an error will be thrown if a property is set to a value of the wrong type.
> - [type.convert] can be used to create a property that will always convert its value to a specific type.
> - [type.maybe] can be  used to create a property that can be `null` or `undefined` as well as whatever valid values the type allows